sql/mob_groups.sql

```diff
@@ -12984,6 +12984,8 @@ INSERT INTO `mob_groups` VALUES (72,2147,206,'Jeumouque_B_DAurphe',0,128,0,0,0,0
 INSERT INTO `mob_groups` VALUES (73,0,206,'Ullegore',0,128,0,0,0,0,0,0);
 INSERT INTO `mob_groups` VALUES (74,0,206,'Mumor',0,128,0,0,0,0,0,0);
 
+INSERT INTO `mob_groups` VALUES (14183,4006,206,'Trion',0,128,0,0,0,75,75,1); -- Trion for rank mission 9-2
```

Comment:
This line was fixed in one of Wren's PRs, so you might have a small conflict here (hopefully not) 🙏

Reply:
I kept the modifications of @wrenffxi